src/editor/CodeHintList.js

```diff
@@ -503,7 +505,16 @@ define(function (require, exports, module) {
                                 "width": hintPos.width + "px"});
         }
     };
-
+    /**
+     * Calls the move up keybind to move hint suggestion selector
+     *
+     * @param {KeyBoardEvent} keyEvent
+     */
+    CodeHintList.prototype.callMoveUp = function (event) {
+        delete event.type;
```

Comment:
I couldn't for some reason just jump to event.type = "keydown" to override the event object property so I deleted it first and this seemed to work.